Composer/packages/client/src/pages/home/Home.tsx

```diff
@@ -163,9 +173,7 @@ const Home: React.FC<RouteComponentProps> = () => {
                 styles={home.newBotItem}
                 title={addButton}
                 onClick={() => {
-                  TelemetryClient.track('CreateNewBotProject', { method: 'newCallToAction' });
-                  setCreationFlowStatus(CreationFlowStatus.NEW);
-                  navigate('projects/create');
+                  onClickNewBot();
```

Comment:
this change will cause the telemetry to incorrectly register clicks on the call to action as clicks on the toolbar.
Maybe pass in a parameter? `onClickNewBot(method)`

Reply:
Good catch, I changed it to call  the telem method from the respective onClicks for the UI components, now OnClickNewBot only calls shared functionality.